github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-postsubmits.yaml

```diff
@@ -1,7 +1,7 @@
 postsubmits:
   kubevirt/cluster-network-addons-operator:
     - name: release-cluster-network-addons-operator
-      run_if_changed: "version/version.go"
+      run_if_changed: "^version/version.go"
```

Comment:
Would you please add the same fix for macvtap-cni? We have the same postsubmit there.

Reply:
#625